app/src/main/java/net/gsantner/markor/format/orgmode/OrgmodeActionButtons.java

```diff
@@ -26,9 +26,11 @@ public List<ActionItem> getFormatActionList() {
                 new ActionItem(R.string.abid_common_checkbox_list, R.drawable.ic_check_box_black_24dp, R.string.check_list),
                 new ActionItem(R.string.abid_common_unordered_list_char, R.drawable.ic_list_black_24dp, R.string.unordered_list),
                 new ActionItem(R.string.abid_common_ordered_list_number, R.drawable.ic_format_list_numbered_black_24dp, R.string.ordered_list),
-                new ActionItem(R.string.abid_common_attach_something, R.drawable.ic_attach_file_black_24dp, R.string.attach),
```

Comment:
The dialog also used to hold a button to insert a color (html foreground, html background or as simple #hexstring). That ones seems to have gone completely down under